server/auth-router.js

```diff
@@ -107,7 +107,7 @@ module.exports = function authRouter(config = {}) {
 		}
 		// Specify ssoRedirect url
 		if (req.query.ssoRedirect) {
-			options.sso_connection = req.query.ssoRedirect;
+			options.ssoRedirect = req.query.ssoRedirect;
```

Comment:
Just out of curiosity, it seems that `login_hint` currently does work or is there is difference in how these two options are handled?

Reply:
right, is this linting or something from the Auth0 docs we didn't catch...

Reply:
Oh it looks like it's because `login_hint` has custom handling in `entry.js`, where the `auth0` PR uses the field directly from the config. We tend to use camel case for parameters, so using this directly seems good.